sapi/apache2handler/php_functions_arginfo.h

```diff
@@ -41,6 +41,7 @@ ZEND_FUNCTION(apache_lookup_uri);
 ZEND_FUNCTION(virtual);
 ZEND_FUNCTION(apache_request_headers);
 ZEND_FUNCTION(apache_response_headers);
+ZEND_FUNCTION(apache_connection_stream);
```

Comment:
See comment on top of this file. This file is generated from the .stub.php file and should not be manually edited.